### DISCUSSION TITLE
Is upscayl cloud going to be floss

### DISCUSSION THREAD

**Discussion by Cuteistfox** [upscayl/upscayl#537] [Q&A]
title says it

**Comment by NayamAmarshe** [comment]
No, unfortunately not because the stack we're using won't be self-hostable (but that also doesn't mean we'll compromise with the privacy policy).

The desktop version however, will continue to receive updates and upgrades :)

**Reply by aaronliu0130** [reply]
Even if it can’t be self-hosted, we can still publish how it works.
